test/erlcloud_ecs_tests.erl

```diff
@@ -81,7 +81,9 @@ operation_test_() ->
       fun update_container_agent_input_tests/1,
       fun update_container_agent_output_tests/1,
       fun update_service_input_tests/1,
-      fun update_service_output_tests/1
+      fun update_service_output_tests/1,
+      fun list_tags_for_resource_input_tests/1,
```

Comment:
can you put these between `list_services_output_tests/1` and `list_task_definition_families_input_tests/1` as well?